_data/finance.yml

```
@@ -383,11 +383,12 @@ websites:
     facebook: VitalityUK
     img: vitality.png
 
-  - name: WaveApps
+  - name: Wave Accounting
```

Wave Accounting seems to be one of the Waveapps products while the login system is for all the products.
Therefore either leave it as it is (`WaveApps`), use their legal name (`Wave Financial`)  or just `Wave`.

---

Didn't realize that, thanks.
```suggestion
  - name: Wave
```